lib/ansible/plugins/filter/ternary.yml

```
@@ -0,0 +1,44 @@
+DOCUMENTATION:
+  name: ternary
+  author: Brian Coca (@bcoca)
+  version_added: '1.9'
+  short_description: Ternary operation filter
+  description:
+    - Return the first value if the input is True, the 2nd if False
+  positional: true_val, false_val
+  options:
+    _input:
+      description: A boolean expression, must evaluate to True or False
+      type: bool
+      required: true
+    true_val:
+      description: Value to return if the input is True
+      type: any
+      required: true
+    false_val:
+      description: Value to return if the input is False
+      type: any
+    none_val:
+      description: Value to return if the input is None, if not set, None will be treated as False
```

```suggestion
      description: Value to return if the input is C(None). If not set, C(None) will be treated as C(False).
```
Doesn't this basically mean `default: false`?